src/Donors/Endpoints/DeleteDonor.php

```diff
@@ -52,15 +52,18 @@ public function registerRoute()
             ]
         );
     }
-    
+
     /**
+     * @unreleased update capability being checked
```

Comment:
Super minor, not really an issue but I think the wording might be slightly off. 

Maybe we should rephrase this to "update validation to align with legacy view" or something along the lines of that.